docs/source/model_doc/pegasus.rst

```diff
@@ -66,6 +66,10 @@ PegasusForConditionalGeneration
 This class inherits all functionality from ``BartForConditionalGeneration``, see that page for method signatures.
 Available models are listed at `Model List <https://huggingface.co/models?search=pegasus>`__
 
+.. autoclass:: transformers.PegasusForConditionalGeneration
```

Comment:
I didn't include this to avoid filling up the page with docstrings that are available elsewhere (Bart).  Can we put it at the bottom of the rst file?

Reply:
Will remove the members, this way we will just see the docstrings of the class, which is not very big (see the Camember doc page for an example).